fe1-web/src/features/wallet/store/DigitalCashStore.ts

```diff
@@ -0,0 +1,29 @@
+import { Hash } from 'core/objects';
+import { getStore } from 'core/redux';
+
+import { TransactionState } from '../objects/transaction';
+import { getDigitalCashState } from '../reducer';
+
+export namespace DigitalCashStore {
```

Comment:
Adding spaces could be nice to improve clarity.

Moreover, I think this namespace lacks some documentation (especially to say that it shouldn't be used outside of a React component).